.github/workflows/build.yml

```diff
@@ -180,8 +180,19 @@ jobs:
         working-directory: ${{github.workspace}}/bin/
         shell: bash
         run: |
-          windeployqt .
+          windeployqt --no-quick-import --no-translations --no-virtualkeyboard .
```

Comment:
`--compiler-runtime`, `--no-webkit2`, `--no-opengl-sw`
Maybe adding `--pdb` and put the pdbs in a separate zip would be helpful for testing

Reply:
pdb doesn't seem very useful as i myseld wouldn't remember which exact artifact i downloaded and neither would the testers